.github/workflows/main.yml

```
@@ -1,39 +1,53 @@
 name: MultiAdmin Build
 
-# Controls when the action will run. Triggers the workflow on push or pull request
-# events but only for the master branch
 on: [push, pull_request]
 
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "build"
   build:
-    # The type of runner that the job will run on
-    runs-on: windows-latest
+    name: .NET ${{matrix.framework}} on ${{matrix.os}}
+    runs-on: ${{matrix.os}}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
```

Use `ubuntu-16.04` instead of `ubuntu-latest` to downgrade the minimum distributive version to run a native app. When I built the app on Ubuntu-20.04 it started crashing on Ubuntu-18.04 with the message `./MultiAdmin: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29’ not found (required by ./MultiAdmin)`, I feel like it's an issue with the native library itself as I haven't found a solution for that. But when I built it on Ubuntu-18.04, it works. .NET 5 supports Ubuntu-16.04 as well.

---

`ubuntu-16.04` is now deprecated, changing to `ubuntu-18.04` instead.
```suggestion
        os: [ubuntu-18.04, windows-latest]
```